content/doc/book/index.html.haml

```diff
@@ -20,6 +20,6 @@ title: "Jenkins Handbook"
               %p{:style => 'font-size: 14px;'}
                 = section.summary
 
-.page-link
+.page-link{:style => 'font-size: 24px;'}
   %a{:href => '/doc'}
-    Up to all documentation
+    Go Back to User Documentation Home
```

Comment:
Capitalization needs to be corrected or the phrase changed to "Return" or "Go to".  The page does not know the navigation history of the user, so I'm more inclined to recommend "Go to User Documentation Home" rather than "Return to User Documentation Home".
```suggestion
    Go back to User Documentation Home
```

Reply:
A preview of the affected page is available at https://deploy-preview-6518--jenkins-io-site-pr.netlify.app/doc/book/